ui/page/wallet_page.go

```diff
@@ -441,7 +462,10 @@ func (pg *WalletPage) walletSection(gtx layout.Context) layout.Dimensions {
 						})
 					}),
 					layout.Rigid(func(gtx C) D {
-						return decredmaterial.Clickable(gtx, pg.listItems[i].addAcctClickable, func(gtx C) D {
+						pg.listLock.Lock()
+						listItem := pg.listItems[i]
+						pg.listLock.Unlock()
```

Comment:
You can use the list item declared earlier here https://github.com/planetdecred/godcr/blob/09cabbcfc0037772251c4648f00dfcf412d6dd81/ui/page/wallet_page.go#L433